tests/Maker/FunctionalTest.php

```diff
@@ -37,7 +37,13 @@ public function testWiring()
 
         $application = new Application($kernel);
         foreach ($finder as $file) {
-            $maker = new \ReflectionClass(sprintf('Symfony\Bundle\MakerBundle\Maker\%s', $file->getBasename('.php')));
+            $namespace = 'Symfony\Bundle\MakerBundle\Maker';
+
+            if (str_ends_with($file->getRelativePath(), 'Security')) {
+                $namespace = 'Symfony\Bundle\MakerBundle\Maker\Security';
+            }
```

Comment:
I think we could find a better way to do this. I believe `$file` should, I think, have some info about its relative path - e.g. `Security/MakerFormLogin.php`. We could use this, change the `/` to `\` and generate the namespace more dynamically.

Reply:
love it